news/13632-fix-conda-env-create-subdir-platform-regression

```diff
@@ -0,0 +1,19 @@
+### Enhancements
```

Comment:
Question: do we track every single issue in this `news` directory? Maybe there's a way to automate this?

Reply:
Typically yes, if the change to the code is significant enough we make sure to point to which pull request a fix was made in via a newsfile so that we can easily included details in our release changelog; I've attempted to automate this via GitHub workflows, but "that way madness lies" (i.e. I tried _lots_ of things, but ended up never being able to auto-generate a newsfile and have it get added to the associated PR)

Reply:
Yes, there are other ways of automating changelog

The most common is for the changelog to be populated from PR titles (or from git commit messages) and using labels to indicate whether to skip certain PRs, we haven't pursued any of these other routes yet as they conflict with how we think about news snippets (e.g., with our current way of doing news snippets, one PR can contain multiple news entries)

See https://github.com/conda/infrastructure/issues/556 for more of our investigations